cli/fortio_main.go

```diff
@@ -207,10 +207,6 @@ func FortioMain(hook bincommon.FortioHook) {
 	scli.ServerMain() // will Exit if there were arguments/flags errors.
 
 	fnet.ChangeMaxPayloadSize(*newMaxPayloadSizeKb * fnet.KILOBYTE)
-	percList, err := stats.ParsePercentiles(*percentilesFlag)
```

Comment:
Not sure if there will be an error if -p flag is used with a command that doesn't use this flag.